Objects/call.c

```diff
@@ -658,6 +657,30 @@ PyEval_CallMethod(PyObject *obj, const char *name, const char *format, ...)
 }
 
 
+PyObject *
+_PyObject_CallMethod(PyObject *obj, PyObject *name,
+                     const char *format, ...)
```

Comment:
Please don't add new API without `PY_SSIZE_T_CLEAN`.
New API should support only Py_ssize_t, even if `PY_SSIZE_T_CLEAN` is not defined.

Ref: https://mail.python.org/archives/list/python-dev@python.org/thread/KSREO43D6GQWO5LMVIU2LF7CP4IBYT2C/#J2D5VBSOQTZOK42AUM5AHPPBA45VNJK4

Reply:
Thanks for pointing this out.

Reply:
fixed